src/background/controller/provider/controller.ts

```diff
@@ -390,7 +391,9 @@ class ProviderController extends BaseController {
     let is1559 = is1559Tx(approvalRes);
     if (
       is1559 &&
-      approvalRes.maxFeePerGas === approvalRes.maxPriorityFeePerGas
+      (approvalRes.maxFeePerGas === approvalRes.maxPriorityFeePerGas ||
+        // list of hardware wallets that do not support EIP-1559
+        [KEYRING_CLASS.HARDWARE.BITBOX02].includes(keyring.type))
```

Comment:
You can simple remove this line https://github.com/RabbyHub/Rabby/blob/develop/src/constant/index.ts#L277 then bitbox02 will disable 1559